### PR TITLE
YOUCODE-456 test changes

### PR DESCRIPTION
**Pull request by youcode-test1**
I did it this way because...

### CODE REVIEW

**Comment by youcode-test2** [comment]
Great job man!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-test",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (youcode-test1) [inline package.json]: I think we should bump the versino to 0.1.1 since this is not a new feature :)

File: src/App.css [hunk 2]
```diff
@@ -1,9 +1,9 @@
 .App {
-  text-align: left;
+  text-align: center;
 }
 
 .App-logo {
-  height: 40vmin;
+  height: 60vmin;
   pointer-events: none;
```
Comment (youcode-test1) [inline src/App.css:7]: Comment!

File: package.json (hunk 1 above)
Comment (youcode-test2) [reply package.json:3]: Thanks for the comment!

File: src/App.css [hunk 3]
```diff
@@ -1,9 +1,9 @@
 .App {
-  text-align: left;
+  text-align: center;
```
Comment (youcode-test2) [inline src/App.css:2]: Wouldn't it be better to align to right?

File: src/App.css [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 }
 
 .App-logo {
-  height: 40vmin;
+  height: 60vmin;
```
Comment (youcode-test2) [inline src/App.css:6]: Are you sure it's better now?

File: src/index.js [hunk 5]
```diff
@@ -4,8 +4,6 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-// test comment
```
Comment (youcode-test2) [inline src/index.js:7]: Nice that you removed it!

**Review by youcode-test2 — APPROVED** [review]
Good job! Just a few simple things.

**Review by youcode-test2 — CHANGES_REQUESTED** [review]
You broke something :) 
Please change this before we can merge this PR.

**Pull request merged**